src/utils/fetch.js

```diff
@@ -28,13 +28,22 @@
  * @license Simplified BSD License
  */
 
-/*
- * Creates URL request path
- */
-const encodeQueryData = data => Object.keys(data)
-  .filter(k => typeof data[k] !== 'object')
-  .map(k => encodeURIComponent(k) + '=' + encodeURIComponent(data[k]))
-  .join('&');
+
+// /*
+//  * Creates URL request path
+//  */
+export const encodeQueryData = (data) => {
+  const replacer = (k, v)=>(v === undefined ? null : v);
+  const pairs = Object.entries(data).map(([key, val]) => {
```

Comment:
Might be better to just filter out any null-ish values before doing the encoding. The backend doesn't really take `null` or `undefined` values anyway.

This way you can just do the following:

```js
const pairs = Object.entries(data)
  .filter(([, val]) => val !== null && val !== undefined)
  .map(([key, val]) => {
    const value = typeof val === 'object' ? JSON.stringify(val) : val;
    return `${encodeURIComponent(key)}=${encodeURIComponent(value)}`;
  });
```